modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx

```diff
@@ -44,7 +44,7 @@ if (Liferay.FeatureFlags['LPS-188645']) {
 	];
 }
 
-if (Liferay.FeatureFlags['LPS-192282']) {
+if (Liferay.FeatureFlags['LPS-164563']) {
```

Comment:
We don't need this as LPS-164563 FF is gating the entire Data Set Manager, so basically we just need to unconditionally add `Actions` to the nav bar items

Reply:
I tried looking into that and couldn't verify, so went with the safe variant. Will fix! Thanks @dsanz